goth/assertions/monitor.py

```diff
@@ -257,15 +259,20 @@ async def _check_assertions(self, events_ended: bool) -> None:
 
             await a.update_events(events_ended=events_ended)
 
+            if a.done:
+                self._logger.debug(
+                    "Assertion '%s' finished after event %s", a.name, event_descr
+                )
+
```

Comment:
Previous assertion success/failure messages that included details of events that made the assertion succeed or fail were a bit unreadable, so I've decided to split them in two messages: one containing the event details and logged with `DEBUG` level, and another one, in color, only stating that the assertion succeeds or fails with a given result, logged as before (usually as `INFO`).